### PR TITLE
[1LP][RFR] Update auth fixtures, views, widgets, and tests

### PR DESCRIPTION
**Pull request by mshriver**
Related to #5754, and includes a hack to work around it.

# REVIEWERS PLEASE NOTE

There will be test failures in PRT, I'm expecting them and am engaging with dev. I will either write many specific RBAC bugs, or start to lump some of the permission mismatches into 2 BZ's that were written while writing these tests.

PRT failures in run 19290 will all be BZ's.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/roles.py [hunk 1]
```diff
@@ -279,3 +279,179 @@ def _remove_from_all(roles, r_page):
         'about'
     ]
 }
+
+# Matches structure/string format of VerticalNavigation output for tree, not UI access control tree
+# TODO include non-vertical nav RBAC to settings, help
+# TODO RBAC goes deeper than veritcal nav, into accordions. example cloud intel -> Reports
```
Comment (mshriver) [inline cfme/roles.py]: This should probably be in a yaml file, I'd prefer to leave that change to a follow-up PR.

**Comment by mshriver** [comment]
I'm removing the logic to modify the role_access data structure based on version within the test. The conditionals are unweildy, and have to take into account Z-stream versions, nested dictionaries, and differences between users. Ultimately I think it's a better implementation to have a separate `role_access` data structure for each z-stream version to avoid confusion and have a more maintainable test - at the cost of very similar data structures with small differences between them.

**Comment by mshriver** [comment]
@psav thoughts on merging this automation re-write of the test? I can skip just based on the 2 BZ's I've written so far, but I'll be writing more BZ's tomorrow morning.

**Pull request merged**